addon/components/model-form/date.hbs

```diff
@@ -2,20 +2,33 @@
   <div class="{{if @horizontal "uk-margin-left uk-flex-last"}}">
     {{component @labelComponent}}
   </div>
-  <PikadayInput
-    @format="DD.MM.YYYY"
-    @placeholder={{t "ember-gwr.components.modelForm.datePlaceholder"}}
-    @onSelection={{@update}}
-    @value={{this.value}}
-    @disabled={{@disabled}}
-    @yearRange={{"2000,currentYear"}}
-    class="uk-width-1 uk-input
-      {{if @isInvalid "uk-form-danger"}}
-      {{if @isValid "uk-form-success"}}"
-    name={{@name}}
-    required={{@required}}
-    ...attributes
-  />
+  {{#if @disabled}}
+    <input
+      type="text"
+      class="uk-input uk-width-1 {{if @isInvalid "uk-form-danger"}} {{if @isValid "uk-form-success"}}"
+      name={{@name}}
+      value={{format-date @value day="2-digit" month="2-digit" year="numeric"}}
+      placeholder={{t "ember-gwr.components.modelForm.datePlaceholder"}}
+      disabled
+      ...attributes
+    >
+  {{else}}
+    <input
+      type="text"
+      class="uk-input uk-width-1 {{if @isInvalid "uk-form-danger"}} {{if @isValid "uk-form-success"}}"
+      name={{@name}}
+      required={{@required}}
+      placeholder={{t "ember-gwr.components.modelForm.datePlaceholder"}}
+      ...attributes
+      {{pikaday
+        value=@value
+        yearRange=this.yearRange
```

Comment:
Where is `this.yearRange` defined?

Reply:
Removed by accident, it's now defined in the component class.